app/build.gradle

```diff
@@ -21,12 +21,12 @@ def getGitHash = { ->
 }
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion sdk_compile_version
```

Comment:
I'd prefer if removed `compileSdkVersion`, `minSdkVersion` and `targetSdkVersion` from the modules, and only set them in the root `build.gradle`. In that case I don't think a variable for those values is needed anymore, since they are defined and used only once.

We should have cleaned those up before I suppose 😄

Reply:
will do